src/Symfony/Component/Mailer/Bridge/Brevo/Transport/BrevoApiTransport.php

```diff
@@ -76,7 +76,7 @@ protected function doSendApi(SentMessage $sentMessage, Email $email, Envelope $e
     /**
      * @return list<array{email: string, name?: string}>
      */
-    protected function stringifyAddresses(array $addresses): array
+    protected function formatAddresses(array $addresses): array
```

Comment:
should be a private method as it does not try to overwrite the parent method anymore.

Reply:
Good point